src/Services/UserManager.vala

```diff
@@ -233,7 +228,55 @@ public class Session.Services.UserManager : Object {
         user_boxes[GUEST_USER_UID].show ();
 
         user_grid.add (user_boxes[GUEST_USER_UID]);
+    }
+
+    // Can't use JUST online users, see comment below.
+    public async int get_n_active_and_online_users () {
+        int n_active_and_online_users = 0;
+
+        if (!manager.is_loaded) {
+            critical ("UserManager not yet loaded");
+            return n_active_and_online_users;
+        }
+
+        foreach (var user in manager.list_users ()) {
+            // Skip system reserved users
+            if (user.uid < RESERVED_UID_RANGE_END || user.uid == NOBODY_USER_UID || user_boxes.has_key (user.uid)) {
```

Comment:
You do no want to skip users that are listed in user_boxes - it stops the tooltip showing other users.